packages/state-transition/src/block/isValidIndexedAttestation.ts

```diff
@@ -44,7 +44,11 @@ export function isValidIndexedAttestationBigint(
  */
 export function isValidIndexedAttestationIndices(state: CachedBeaconStateAllForks, indices: number[]): boolean {
   // verify max number of indices
-  if (!(indices.length > 0 && indices.length <= MAX_VALIDATORS_PER_COMMITTEE)) {
+  const maxIndices =
+    state.fork.epoch >= state.config.ELECTRA_FORK_EPOCH
```

Comment:
I think the Lodestar convention of checking whether we are electra-activated is `state.config.getForkSeq(state.slot) >= ForkSeq.electra`. Would be nice if we can keep this consistent.